docs/reference/commandline/tag.md

```diff
@@ -16,5 +16,11 @@ parent = "smn_cli"
 
       --help               Print usage
 
+Image name must match the regular expression `[a-z0-9]+`. 
```

Comment:
I think it's better not to include `[USERNAME/]`, because this isn't actually part of the reference grammar; it's just convention. It could be confusing for these docs to diverge from the actual grammar.